src/utils/withFilters.js

```diff
@@ -12,32 +12,59 @@ function withFilters(apiName, initialFilters, initialSort) {
         this.actions = {
```

Comment:
do we need a withFilter while we can have it in state and passing it too hooks ?